config/peerpods/podvm/lib.sh

```diff
@@ -198,8 +198,12 @@ function prepare_source_code() {
 
     if [[ "$CONFIDENTIAL_COMPUTE_ENABLED" == "yes" ]]; then
         sed 's/default SetPolicyRequest := true/default SetPolicyRequest := false/; s/default ExecProcessRequest := true/default ExecProcessRequest := false/' \
-            "${podvm_dir}"/files/etc/kata-opa/default-policy.rego > "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
-        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego  "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
+            "${podvm_dir}"/files/etc/kata-opa/default-policy.rego >"${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego
+        ln -sf "${podvm_dir}"/files/etc/kata-opa/coco-default-policy.rego "${podvm_dir}"/files/etc/kata-opa/default-policy.rego
```

Comment:
These are just dropping some extra space characters, right ?

Reply:
Right.. Will wait for https://github.com/openshift/sandboxed-containers-operator/pull/434 to get merge and rebase this one.